crates/sui-types/src/base_types.rs

```diff
@@ -642,6 +642,10 @@ impl SequenceNumber {
     pub const MIN: SequenceNumber = SequenceNumber(u64::MIN);
     pub const MAX: SequenceNumber = SequenceNumber(0x7fff_ffff_ffff_ffff);
 
+    // Wrapped objects that are then deleted will be set to MAX, so cap sequence
+    // numbers for other objects at MAX - 1.
+    pub const MAX_NOT_WRAPPED_AND_DELETED: SequenceNumber = SequenceNumber(Self::MAX.0 - 1);
+
```

Comment:
I think we can just rename this to `MAX_INPUT_VERSION`? Its the maximum for an input, because we need to be able to increment it for it's output

Reply:
If your above message is correct though, can we not get rid of it entirely, and phrase the check it was used for as `sequence_number < SequenceNumber::MAX`?

Reply:
... I think you are right lol